internal/perms/acl.go

```diff
@@ -71,7 +71,7 @@ func NewACL(grants ...Grant) ACL {
 }
 
 // Allowed determines if the grants for an ACL allow an action for a resource.
-func (a ACL) Allowed(r Resource, aType action.Type) (results ACLResults) {
+func (a ACL) Allowed(r Resource, aType action.Type, isAnonUser bool) (results ACLResults) {
```

Comment:
perhaps require the userId here (vs the bool) and then support an option like WithAllowAnon(bool) to support calls like perms.Parse(...) where you don't want to enforce the u_anon rules?